### PR TITLE
Improve project-template

### PR DESCRIPTION
**Pull request by niklashhh**

### CODE REVIEW

File: .env [hunk 1]
```diff
@@ -1 +1 @@
-ENDPOINT=http://0.0.0.0:9000
+BACKEND_PORT=9000
```
Comment (mrako) [inline .env]: Remove .env

File: .gitignore [hunk 2]
```diff
@@ -1,8 +1,49 @@
...
+
+# Sqlite
 *.sqlite
+*.sqlite3
+
+# ides
+*.idea
+*.iml
 
 # macOS
```
Comment (mrako) [inline .gitignore]: Remove this

File: .gitignore [hunk 3]
```diff
@@ -1,8 +1,49 @@
...
+# OSX
+.DS_Store
+
+# Node
+node_modules
+
+# Logs
+*.log
+
+# Ansible
```
Comment (mrako) [inline .gitignore]: This is probably not needed

File: README.md [hunk 4]
```diff
@@ -1,181 +1,129 @@
...
-docker-compose up
+> If you work in a group, only one person needs to fork the repository.
+> The others need to be members of the repository. See [collaborating with other users](#collaborating)
 
-Hit Ctrl+C to stop the process
-```
+Then clone the forked repository using SSH
 
-What if I want to run the project in the background?
+    git clone git@github.com:<user>/koodi101-template.git
```
Comment (mrako) [inline README.md]: The project name has changed

File: README.md [hunk 5]
```diff
@@ -1,181 +1,129 @@
...
-Nice, how can I now see if the project is already running?
+> Note that for the _environment variables_ to load correctly, you can't be in a subfolder of koodi101-template
 
-```shell
-docker-compose ps
-```
+Then build the docker images and start the services
 
-Cool, how can I stop the project from running and start from scratch
+    docker-compose build
```
Comment (mrako) [inline README.md]: Rather use ```shell or ```bash

File: backend/Dockerfile [hunk 6]
```diff
@@ -1,4 +1,6 @@
-FROM node:8.1.2-alpine
+FROM node:13.7.0-alpine
+
+EXPOSE 9000
```
Comment (mrako) [inline backend/Dockerfile]: I don't think expose is needed anymore

File: backend/Dockerfile [hunk 7]
```diff
@@ -1,4 +1,6 @@
-FROM node:8.1.2-alpine
+FROM node:13.7.0-alpine
```
Comment (mrako) [inline backend/Dockerfile]: rather latest instead of a specific version

File: backend/src/app.js [hunk 8]
```diff
@@ -8,7 +8,8 @@ const chats = require('./controllers/chats');
 const greeting = require('./controllers/greeting');
 const sensors = require('./controllers/sensors');
 
-const app = module.exports = new Koa();
+const app = new Koa();
+module.exports = app;
```
Comment (mrako) [inline backend/src/app.js]: module exports should be last row.

File: backend/src/controllers/chats.js [hunk 9]
```diff
@@ -16,8 +15,8 @@ exports.list = async (ctx) => {
 exports.create = async (ctx) => {
   const params = ctx.request.body;
 
-  const chat = await database.Chat.create({message: params.message});
+  const chat = await database.Chat.create({ message: params.message });
 
-  ctx.body = await chat.toJSON();
+  ctx.body = chat;
```
Comment (mrako) [inline backend/src/controllers/chats.js]: this looks suspicious as a read code (chat is an object and setting it to body might be misleading to a reader)

File: backend/src/models/sensor.js [hunk 10]
```diff
@@ -1,23 +1,24 @@
-var Sequelize = require('sequelize');
-var sequelize = require('./sequelize');
+const Sequelize = require('sequelize');
+const sequelize = require('./sequelize');
 
-var SensorEntry = sequelize.define('sensor', {
+const SensorEntry = sequelize.define('sensor_entries', {
```
Comment (mrako) [inline backend/src/models/sensor.js]: Rather just event?

File: frontend/.babelrc [hunk 11]
```diff
@@ -0,0 +1,13 @@
+{
+  "presets": [
+    [
+      "@babel/preset-env",
```
Comment (mrako) [inline frontend/.babelrc]: This is also due to get obsolete. Rather avoid if possible.

File: frontend/package.json [hunk 12]
```diff
@@ -1,32 +1,38 @@
 {
-  "name": "react-tictactoe",
+  "name": "koodi101-template-frontend",
```
Comment (mrako) [inline frontend/package.json]: Rename

File: frontend/package.json [hunk 13]
```diff
@@ -1,32 +1,38 @@
...
   "version": "0.0.1",
-  "description": "Koodi101 React TicTacToe",
+  "description": "Koodi101-Template simple frontend",
   "author": "Marko Klemetti <marko.klemetti@gmail.com>",
   "license": "MIT",
   "repository": {
     "type": "git",
-    "url": "https://github.com/koodi101/react-tictactoe.git"
+    "url": "https://github.com/koodi101/koodi101-template.git"
```
Comment (mrako) [inline frontend/package.json]: Rename

File: backend/src/controllers/chats.js (hunk 9 above)
Comment (niklashhh) [reply backend/src/controllers/chats.js]: I think it makes sense that the chat created object is returned in the response body

File: frontend/.babelrc (hunk 11 above)
Comment (niklashhh) [reply frontend/.babelrc]: Which part? Couldn't find anything online.

File: backend/src/controllers/events.js [hunk 14]
```diff
@@ -0,0 +1,25 @@
...
+    results: events,
+  };
+
+  ctx.body = response;
+};
+
+exports.create = async (ctx) => {
+  const params = ctx.request.body;
+
+  const sensor = await database.SensorEntry.create({
```
Comment (mrako) [inline backend/src/controllers/events.js]: I'm not sure if there is a sensorentry anymore.

File: backend/src/controllers/chats.js [hunk 15]
```diff
@@ -16,8 +15,8 @@ exports.list = async (ctx) => {
 exports.create = async (ctx) => {
   const params = ctx.request.body;
 
-  const chat = await database.Chat.create({message: params.message});
+  const chat = await database.Chat.create({ message: params.message });
```
Comment (mrako) [inline backend/src/controllers/chats.js]: Do we need both chat and event in here?

Comment (niklashhh) [reply backend/src/controllers/chats.js]: I think it's fine that chat remains as it was also in lesson3-nodejs-template

**Pull request merged**